views/includes/metaFooterJS.njk

```diff
@@ -1,10 +1,4 @@
-<script src="https://www.google-analytics.com/cx/api.js"></script>
```

Comment:
Removing this as we're not using the google experiments / ab testing code anymore so we can avoid loading all this.